src/components/Results/Results.tsx

```diff
@@ -89,8 +89,8 @@ const Results: React.FC<Props> = ({ measureFile, patientFile, htmls }) => {
               )}
             </Grid>
             {results && <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />}
-            {patientFile && <h2>Patient Bundle:</h2>}
-            {patientFile && (
+            {patientFile && outputType !== 'dataRequirement' && <h2>Patient Bundle:</h2>}
+            {patientFile && outputType !== 'dataRequirement' && (
```

Comment:
Since the condition is the same, this could be shortened to just one conditional and a larger tsx body surrounded by <> </>

Same with on lines 75 & 75 above.